web-components/src/components/card/tokens/lm-card-token.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+
+const colors = require("@momentum-ui/tokens/dist/colors.json");
+
+const card = {
+  prefix: "lm",
+  component: "card",
+  color: {
+    light: colors.gray[70].name,
+    dark: colors.gray[40].name,
+    title: {
+      light: colors.gray[100].name,
+      dark: colors.gray["05"].name
+    }
+  },
+  bg: {
+    light: colors.white[100].name,
+    dark: colors.gray[90].name
+  },
+  border: {
+    light: colors.gray[20].name,
+    dark: colors.gray[80].name
+  }
```

Comment:
this is arguable, we do have a global `--md-quaternary-bg-color` that has these hues, but it says `bg` color not `border` color, so semantically the custom token is more correct. Probably OK to leave this one as is.